changelog/unreleased/features/6943.md

```diff
@@ -0,0 +1,5 @@
+- Added `RoadObjectType.IC` and `RoadObjectType.JCT` for interchanges and junctions and the corresponding classes `Interchange` and `Junction`.
+- Limited number of reported alternatives to 2.
+- Sped up onboard routing cancellation.
+- Fixed null pointer dereference.
+- Fixed unexpected NaN values in location data.
```

Comment:
This should all be in improvements changelog section, not features.

Reply:
Oops, I forgot we had different directories...